routes/game.js

```diff
@@ -192,9 +192,12 @@ routes.init = function(req, res) {
           console.error("Error saving new card", err);
           res.status(500).send("Error saving new card");
         }
+
         res.send({
           user: req.session.user,
           game: data,
+          currentUser: req.session.user,
```

Comment:
We're sending `req.session.user` two lines above this. Why are we sending it again under a different name?